notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>I completely understand that this may just a me thing, but I prefer to use "horizontal axis" instead of "x-axis" and "vertical axis" instaed of "y-axis." At the undergraduate level students can stuggle with variables that are not "x" being on the horizontal axis, and I think this tutorial is definitely accessible to them!</p><p> </p><blockquote>So, we are going need a variable for:</blockquote><blockquote>array of times of all observations</blockquote><p> </p><p>So, we are going to need an array for: times of all observations, etc. Then it might be helpful to name the variables with each bullet point. </p><p> </p><blockquote>quieried</blockquote><p> </p><p>queried</p><p> </p><blockquote>observation, for the</blockquote><p> </p><p>Split into 2 sentences instead of comma.</p><p> </p><p>The modifications were very clearly explained!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>